cloud/amazon/iam_policy.py

```diff
@@ -283,7 +283,7 @@ def main():
       iam_name=dict(default=None, required=False),
       policy_name=dict(default=None, required=True),
       policy_document=dict(default=None, required=False),
-      policy_json=dict(default=None, required=False),
+      policy_json=dict(type='dict', default=None, required=False),
```

Comment:
using a template to read a json file actually produces a dict, but type=json should be able to handle both and produce a json string for the module either way.